plugin-dev/Source/Sentry/Private/Desktop/CrashReporter/SentryCrashContext.cpp

```diff
@@ -31,15 +31,15 @@ void FSentryCrashContext::Apply(TSharedPtr<SentryScopeDesktop> Scope)
 {
 	const USentrySettings* Settings = FSentryModule::Get().GetSettings();
 
-	const FSessionContext& SessionContext = CrashContext.SessionContext;
+	const FSessionContext& SessionContext = CrashContext->SessionContext;
 
-	Scope->SetExtraValue("Crash Type", FGenericCrashContext::GetCrashTypeString(CrashContext.CrashType));
-	Scope->SetExtraValue("IsEnsure", CrashContext.CrashType == ECrashContextType::Ensure ? TEXT("true") : TEXT("false"));
+	Scope->SetExtraValue("Crash Type", FGenericCrashContext::GetCrashTypeString(CrashContext->CrashType));
```

Comment:
Worth noting that we should consider moving all the things that are getting set as `Extra` into their own context.

Reply:
Good point, let's track this in #411